pkg/server/admin.go

```diff
@@ -3990,3 +3990,35 @@ func (s *adminServer) SetTraceRecordingType(
 	})
 	return &serverpb.SetTraceRecordingTypeResponse{}, nil
 }
+
+func (s *adminServer) RecoveryCollectReplicaInfo(
+	request *serverpb.RecoveryCollectReplicaInfoRequest,
+	server serverpb.Admin_RecoveryCollectReplicaInfoServer,
+) error {
+	panic("To be implemented by #93040")
```

Comment:
nit: return `errors.AssertionFailedf`, otherwise this is a trivial DoS vector.

Reply:
On second thought, maybe we have a panic handler that would catch it. Either way, `AssertionFailedf` is more appropriate when we have an error return path.